scipy/stats/_sensitivity_analysis.py

```diff
@@ -93,12 +93,20 @@ def sample_A_B(
 def sample_AB(A: np.ndarray, B: np.ndarray) -> np.ndarray:
     """AB matrix.
 
-    AB: rows of B into A. Shape (d, d, n).
+    AB: rows of B into A. Shape (d, d, n). See sec. 3 from [1]_.
     - Copy A into d "pages"
     - In the first page, replace 1st rows of A with 1st row of B.
     ...
     - In the dth page, replace dth row of A with dth row of B.
     - return the stack of pages
+
+    References
```

Comment:
Please remove these changes. It's redundant with the main function and this function is private.